docker/build_support/compose

```diff
@@ -23,8 +23,8 @@ export GROUP_ID=$(id -g)
 
 cp $DIR/docker-compose.shared.yml /tmp/composed.yml
 for compose_yml in $@; do
-    docker-compose -f $compose_yml config > /tmp/next.yml
-    docker-compose -f /tmp/next.yml -f /tmp/composed.yml config > /tmp/composed.yml.next
+    docker-compose -f $compose_yml config --no-interpolate > /tmp/next.yml
```

Comment:
Because the Vulcan image requires `MONOETNA_SHA` to be an environment variable, when that gets composed into the overall `docker/docker-compose.yml`, if it is interpolated then it becomes static and won't update unless the `docker/docker-compose.yml` file is rebuilt. I think this file only gets rebuilt if any of the app `docker-compose.yml` files changes, so code changes or new commits wouldn't necessarily update this value. But, if we don't interpolate at this step, in `docker/docker-compose.yml` it remains as a reference to the environment variable, and subsequent `make up`, `make restart`, etc., calls will include an updated SHA.